app/lib/test_applications.rb

```diff
@@ -132,34 +135,44 @@ def create_application_to_courses(
     end
 
     Audited.audit_class.as_user(candidate) do
-      traits = [%i[with_safeguarding_issues_disclosed
-                   with_safeguarding_issues_never_asked
-                   minimum_info]].sample
+      traits = if duplicate_candidates
+                 [%i[with_safeguarding_issues_disclosed
+                     with_safeguarding_issues_never_asked
+                     minimum_info duplicate_candidates]].sample
```

Comment:
I hadn't really thought about putting the trait into this sample set, when we talked earlier, but perhaps it's a great way to get a more random set of duplicates.

However with only four chances your chances of getting a duplicate are not all that high. Maybe we could just do away with the `duplicate_candidates` parameter and always  use the trait here. What do you think?